### ISSUE TITLE
Add option to enable/disable loading spinner when running a query

### ISSUE THREAD

**Issue by markusstephanides** [openblocks-dev/openblocks#276]
Currently all components that depend data from queries (non-JS queries afaik) show a loading spinner during the required data is loaded. This behaviour might not be wanted be the user in all cases. 

So it might make sense to create a toggle on either the query or the component you want to toggle it on.

What do you think?

**Comment by YoJayz** [comment]
Hi @markusstephanides 

For **Table** component, you can toggle off "Show spinner during data loading" in **Interaction** section of **Properties** pane. Can this be helpful to you?

![image](https://user-images.githubusercontent.com/113571084/220585681-2f463c49-0b1a-4c6b-8e9b-cb6992348039.png)

As for other components and queries, it seems that there isn't a workaround now. We'll consider how to solve it, and we are glad to hear your suggestions and see your contribution.

**Comment by markusstephanides** [comment]
Cool, unfortunately I'm not using the table component. 

What would you think of having this toggle in the other components as well?

**Comment by MichalisDBA** [comment]
Yes that would be great to enable / disable the spinner on all other components!

**Comment by YoJayz** [comment]
Hi all, thanks for raising and supporting this feature request.
We are planning to work on this in the next month. Please stay tuned and thanks for your patience.
